config/locales/es.yml

```diff
@@ -83,6 +83,7 @@ es:
   upload_new_file: "Arrastre el archivo aquí, o haga clic para agregar."
   reject_new_file: "Tipo de archivo no soportado."
   accepted_tooltip: "La tarea ya ha sido aceptada. No se puede modificar."
+  revert_tooltip: "Invertir el orden de las columnas."
```

Comment:
We could name it `revert_columns_tooltip` to make it clearer, what do you think?

Reply:
Nice! I think that make sense.